### ISSUE TITLE
Request: Multi-Zone lighting control for Miniled Flow X16 

### ISSUE THREAD

**Issue by ryangodburn** [JamesCJ60/Armoury-Control#3]
Hello, I was wondering if the feature from armoury crate can be implemented here for Miniled Flow X16 which is a toggle when in SDR mode that switches from multi-zone backlighting on the screen to single-zone lighting and vice-versa? This is my only major missing feature for me with armoury control now compared to Asus's bloatware.
![image](https://user-images.githubusercontent.com/44278185/222908154-cba05298-1baa-4539-ac6f-739e7cf483c4.png)
![image](https://user-images.githubusercontent.com/44278185/222908171-de109ef1-f6bf-44a2-b0cd-14647b25c850.png)
This works automatically on and off when HDR is turned on and off, but I am looking for control while in SDR.

**Comment by RomanYazvinsky** [comment]
@ryangodburn Hi, you can help us!
Here's the script - [scan-atkacpi.zip](https://github.com/JamesCJ60/Armoury-Control/files/10902569/scan-atkacpi.zip).
This is modification of [G-helper Diagnostics script](https://github.com/seerge/g-helper/issues/38#issuecomment-1452142709).

1. Run this script as Admin
2. Copy results to notepad or here
3. Change the setting in Armoury Crate
4. Run script again
5. Copy new report and send us both reports

Long story short, we need to find the difference between those two reports. I guess something with id starting from 0x500xx will change from 0 to 1

**Comment by ryangodburn** [comment]
Multizone OFF:
50019 : 0 (0)                                                                                                           5001e : 0 (0)                                                                                                           50020 : 1 (1)                                                                                                           60023 : ffffffffffff0004 (-65532)                                                                                       60024 : a36c8086 (2741796998)                                                                                           60026 : 0 (0)                                                                                                           6005b : 4 (4)                                                                                                           60061 : ffffffffffff0000 (-65536)                                                                                       60077 : 0 (0)                                                                                                           60094 : 6 (6)                                                                                                           80041 : ffffffffffff0000 (-65536)                                                                                       80042 : 0 (0)                                                                                                           80043 : ffffffffffff0000 (-65536)                                                                                       80044 : ffffffffffff0000 (-65536)                                                                                       90016 : 1 (1)                                                                                                           90017 : 0 (0)                                                                                                           90018 : 0 (0)                                                                                                           90019 : 0 (0)                                                                                                           9001c : 3 (3)                                                                                                           90020 : 1 (1)                                                                                                           90030 : 40 (64)                                                                                                         100051 : 1 (1)                                                                                                          110013 : 0 (0)                                                                                                          110014 : 0 (0)                                                                                                          110015 : ffffffffffff0000 (-65536)                                                                                      110016 : ffffffffffff0000 (-65536)                                                                                      110022 : 99 (153)                                                                                                       110023 : d7 (215)                                                                                                       Invoke-CimMethod : Generic failure                                                                                      At C:\Users\Ryan\Desktop\scan-atkacpi.ps1:5 char:20                                                                     + ...  $result = (Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -Cl ...                                         +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                 + CategoryInfo          : NotSpecified: (AsusAtkWmi_WMNB...PNP0C14\ATK_0"):CimInstance) [Invoke-CimMethod], CimExc     eption                                                                                                                   + FullyQualifiedErrorId : HRESULT 0x80041001,Microsoft.Management.Infrastructure.CimCmdlets.InvokeCimMethodCommand                                                                                                                          110024 : ffff0000 (-65536)                                                                                              Invoke-CimMethod : Generic failure                                                                                      At C:\Users\Ryan\Desktop\scan-atkacpi.ps1:5 char:20
+ ...  $result = (Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -Cl ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (AsusAtkWmi_WMNB...PNP0C14\ATK_0"):CimInstance) [Invoke-CimMethod], CimExc
   eption
    + FullyQualifiedErrorId : HRESULT 0x80041001,Microsoft.Management.Infrastructure.CimCmdlets.InvokeCimMethodCommand

110025 : ffff0000 (-65536)
110026 : ffffffffffff0000 (-65536)
110027 : ffffffffffff0000 (-65536)
110031 : 0 (0)
Invoke-CimMethod : Generic failure
At C:\Users\Ryan\Desktop\scan-atkacpi.ps1:5 char:20
+ ...  $result = (Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -Cl ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (AsusAtkWmi_WMNB...PNP0C14\ATK_0"):CimInstance) [Invoke-CimMethod], CimExc
   eption
    + FullyQualifiedErrorId : HRESULT 0x80041001,Microsoft.Management.Infrastructure.CimCmdlets.InvokeCimMethodCommand

110032 : ffff0000 (-65536)
110033 : ffffffffffff0000 (-65536)
120057 : 80000 (524288)
120061 : 20000 (131072)
12006c : 0 (0)
120075 : 3 (3)
120079 : 2f8 (760)
12007a : 0 (0)
120093 : 50 (80)
120094 : 35 (53)
120097 : 2c (44)
1200a0 : 0 (0)
1200a1 : 0 (0)
1200a2 : 0 (0)
1200a3 : 0 (0)
1200b0 : 0 (0)
1200b1 : 0 (0)
1200c0 : 0 (0)
1200c1 : 0 (0)
1200c2 : 0 (0)
130021 : 2 (2)
130022 : 1 (1)
130026 : ffffffffffff0000 (-65536)
130031 : 1 (1)

Multizone ON: 
50019 : 0 (0)
5001e : 1 (1)                                                                                                           50020 : 1 (1)                                                                                                           60023 : ffffffffffff0004 (-65532)                                                                                       60024 : a36c8086 (2741796998)                                                                                           60026 : 0 (0)                                                                                                           6005b : 4 (4)                                                                                                           60061 : ffffffffffff0000 (-65536)                                                                                       60077 : 0 (0)                                                                                                           60094 : 6 (6)                                                                                                           80041 : ffffffffffff0000 (-65536)                                                                                       80042 : 0 (0)                                                                                                           80043 : ffffffffffff0000 (-65536)                                                                                       80044 : ffffffffffff0000 (-65536)                                                                                       90016 : 1 (1)                                                                                                           90017 : 0 (0)                                                                                                           90018 : 0 (0)                                                                                                           90019 : 0 (0)                                                                                                           9001c : 3 (3)                                                                                                           90020 : 1 (1)                                                                                                           90030 : 40 (64)                                                                                                         100051 : 1 (1)                                                                                                          110013 : 0 (0)                                                                                                          110014 : 0 (0)                                                                                                          110015 : ffffffffffff0000 (-65536)                                                                                      110016 : ffffffffffff0000 (-65536)                                                                                      110022 : 99 (153)                                                                                                       110023 : d7 (215)                                                                                                       Invoke-CimMethod : Generic failure                                                                                      At C:\Users\Ryan\Desktop\scan-atkacpi.ps1:5 char:20                                                                     + ...  $result = (Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -Cl ...                                         +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                 + CategoryInfo          : NotSpecified: (AsusAtkWmi_WMNB...PNP0C14\ATK_0"):CimInstance) [Invoke-CimMethod], CimExc     eption                                                                                                                   + FullyQualifiedErrorId : HRESULT 0x80041001,Microsoft.Management.Infrastructure.CimCmdlets.InvokeCimMethodCommand                                                                                                                          110024 : ffff0000 (-65536)                                                                                              Invoke-CimMethod : Generic failure                                                                                      At C:\Users\Ryan\Desktop\scan-atkacpi.ps1:5 char:20
+ ...  $result = (Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -Cl ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (AsusAtkWmi_WMNB...PNP0C14\ATK_0"):CimInstance) [Invoke-CimMethod], CimExc
   eption
    + FullyQualifiedErrorId : HRESULT 0x80041001,Microsoft.Management.Infrastructure.CimCmdlets.InvokeCimMethodCommand

110025 : ffff0000 (-65536)
110026 : ffffffffffff0000 (-65536)
110027 : ffffffffffff0000 (-65536)
110031 : 0 (0)
Invoke-CimMethod : Generic failure
At C:\Users\Ryan\Desktop\scan-atkacpi.ps1:5 char:20
+ ...  $result = (Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -Cl ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (AsusAtkWmi_WMNB...PNP0C14\ATK_0"):CimInstance) [Invoke-CimMethod], CimExc
   eption
    + FullyQualifiedErrorId : HRESULT 0x80041001,Microsoft.Management.Infrastructure.CimCmdlets.InvokeCimMethodCommand

110032 : ffff0000 (-65536)
110033 : ffffffffffff0000 (-65536)
120057 : 80000 (524288)
120061 : 20000 (131072)
12006c : 0 (0)
120075 : 3 (3)
120079 : 30c (780)
12007a : 0 (0)
120093 : 50 (80)
120094 : 36 (54)
120097 : 2e (46)
1200a0 : 0 (0)
1200a1 : 0 (0)
1200a2 : 0 (0)
1200a3 : 0 (0)
1200b0 : 0 (0)
1200b1 : 0 (0)
1200c0 : 0 (0)
1200c1 : 0 (0)
1200c2 : 0 (0)
130021 : 2 (2)
130022 : 1 (1)
130026 : ffffffffffff0000 (-65536)
130031 : 1 (1)

**Comment by ryangodburn** [comment]
This might be easier to r
[Results.txt](https://github.com/JamesCJ60/Armoury-Control/files/10902660/Results.txt)
ead:

**Comment by ryangodburn** [comment]
No problem. Can't wait to see if something can come from this!

**Comment by JamesCJ60** [comment]
Hi @ryangodburn, I have just added this feature to Armoury Control. Thanks for running the script to find the ACPI command ID!

I'm not going to release an official update just yet, so here you go: [Armoury Control.zip](https://github.com/JamesCJ60/Armoury-Control/files/10904123/Armoury.Control.zip)

**Comment by ryangodburn** [comment]
It seems to be working perfectly, thank you!

**Comment by ryangodburn** [comment]
Also it should be locked in multi-zone mode when HDR is on.
